File: core-java-program/gcr-codebase/programming-elements/CelToFah.java
import java.util.Scanner;
public class CelToFah{
	public static void main(String[] args){
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter the temperature in Celsium: ");
		double cel = sc.nextDouble();
		double fah = (cel*9.0/5.0)+32;
		System.out.println("Temperature in Fahrenheit: " + fah);
	}
}

File: core-csharp-program/gcr-codebase/programming-elements/CelToFah.cs
using System;

public class CelToFah
{
    public static void Main(string[] args)
    {
        Console.Write("Enter the temperature in Celsius: ");
        double cel = Convert.ToDouble(Console.ReadLine());
        double fah = (cel * 9.0 / 5.0) + 32;
        Console.WriteLine("Temperature in Fahrenheit: " + fah);
    }
}
